dim-database/src/episode.rs

```diff
@@ -248,7 +248,7 @@ impl Episode {
             season_number,
             season_number
         )
-        .fetch_one(&mut *conn)
+        .fetch_one(conn.as_mut())
```

Comment:
I feel like these changes are mostly pedantic.

Reply:
I will try to see if it can be done in another way

Reply:
Because Excutor trait is no longer impl for Transaction. 

Some option:
1. 
I see another pull request https://github.com/Dusk-Labs/dim/pull/597 which use extra level of deref like this
```
-            .fetch_all(&mut *conn)
+           .fetch_all(&mut **conn)
```

2.
or we implement excutor back for Transaction.

which way do you prefer

Reply:
The reason there isn't a implementation of `Executor` for `Transaction` in sqlx now is that it couldn't exist. The changelog of sqlx explicitly said that dereferencing the `Transaction` is the way to do it now, so not sure how option 2 is going to work?

Reply:
> The reason there isn't a implementation of `Executor` for `Transaction` in sqlx now is that it couldn't exist. The changelog of sqlx explicitly said that dereferencing the `Transaction` is the way to do it now, so not sure how option 2 is going to work?

ha, seems so.